packages/idafree.vm/tools/chocolateyinstall.ps1

```diff
@@ -30,16 +30,23 @@ try {
     Remove-Item $desktopShortcut -Force -ea 0
   }
 
+  # Download ida_launcher.exe to assist with taskbar and right click option and store it in %RAW_TOOLS_DIR%
+  # ida_launcher.exe is a custom binary that searches for the latest ida64.exe and executes it
+  $launcherName = 'ida_launcher'
+  $launcherSource = 'https://raw.githubusercontent.com/mandiant/vm-packages/main/ida_launcher/ida_launcher.exe'
```

Comment:
This is what I meant with using a hash commit:
```suggestion
  $launcherSource = 'https://github.com/mandiant/VM-Packages/blob/6416814e4c538c79066c7c830b5c7f9ebcea8be8/ida_launcher/ida_launcher.exe'
```

Reply:
I had to keep the `raw.githubusercontent` piece, but I was able to convert it to a specific commit style.

I'm not sure how that might effect things later down the road if the launcher ever gets updated, since it will always pull from this commit instead of from the main branch.

Reply:
The will have to update it using two PR like we did this time. But that's the only way to ensure we are testing the correct version. So I think it is perfect like this 😉